pkgs/development/libraries/fstrcmp/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchzip, libtool, ghostscript, groff }:
+
+stdenv.mkDerivation rec {
+  name = "fstrcmp-${version}";
+  version = "0.7";
+
+  src = fetchzip {
+    url = "https://sourceforge.net/projects/fstrcmp/files/fstrcmp/${version}/fstrcmp-${version}.D001.tar.gz";
+    sha256 = "0yg3y3k0wz50gmhgigfi2dx725w1gc8snb95ih7vpcnj6kabgz9a";
+  };
+
+  outputs = [ "out" "dev" "doc" "man" "devman" ];
+
+  buildInputs = [ libtool ghostscript groff ];
```

Comment:
I take it all of these are used strictly during the build? If so they can be `nativeBuildInputs`

Reply:
Correct, fixed.